qtpy/QtCore.py

```diff
@@ -38,6 +38,15 @@
     from .enums_compat import promote_enums
     promote_enums(QtCore)
     del QtCore
+
+    # Map missing unscoped access enum values
+    Qt.BackButton = Qt.XButton1
+    Qt.ForwardButton = Qt.XButton2
+
+    # Map Qt6 removed obsolete ItemDataRole enum values
```

Comment:
```suggestion
    # Alias deprecated ItemDataRole enum values removed in Qt6
```

Clarify this a bit, since the current syntax could be rather confusing